src/main/scala/ai/privado/exporter/ConsoleExporter.scala

```diff
@@ -0,0 +1,138 @@
+        if (collectionsSourceMap.contains(collect.sourceId)) {
+          collectionsSourceMap(collect.sourceId).addOne(collection.name)
+        } else {
+          collectionsSourceMap.addOne(collect.sourceId -> mutable.Set[String](collection.name))
+        }
+      })
+    })
+
+    // Storages - SourceId Map
+    dataflowsOutput("storages").foreach(storage => {
+      val storages = mutable.Set[String]()
+      storage.sinks.foreach(sink => {
+        storages.addOne(sink.name)
+      })
+      storageSourceMap.addOne(storage.sourceId -> storages)
+    })
+
+    // Third Parties - SourceId Map
+    dataflowsOutput("third_parties").foreach(thirdParty => {
+      val thirdParties = mutable.Set[String]()
+      thirdParty.sinks.foreach(sink => {
+        if (sink.apiUrl.size > 0) {
+          sink.apiUrl.foreach(urlString => {
+            val url = new URL("https://" + urlString.replaceAll("https://", "").trim)
+            thirdParties.addOne(url.getHost.replaceAll("www.", ""))
+            uniqueThirdParties.addOne(url.getHost.replaceAll("www.", ""))
+          })
+        } else {
+          thirdParties.addOne(sink.name)
+          uniqueThirdParties.addOne((sink.name))
+        }
+      })
+      thirdPartySourceMap.addOne(thirdParty.sourceId -> thirdParties)
+    })
+
+    // Internal APIs - SourceId Map
+    dataflowsOutput("internal_apis").foreach(internalAPI => {
+      val internalAPIs = mutable.Set[String]()
+      internalAPI.sinks.foreach(sink => {
+        if (sink.apiUrl.size > 0) {
+          sink.apiUrl.foreach(urlString => {
+            val url = new URL("https://" + urlString.replaceAll("https://", "").trim)
+            internalAPIs.addOne(url.getHost.replaceAll("www.", ""))
+          })
+        }
+      })
+      internalAPIsSourceMap.addOne(internalAPI.sourceId -> internalAPIs)
+    })
+
+    println("\n-----------------------------------------------------------")
+    println("SUMMARY")
+    println("-----------------------------------------------------------")
+    println("\nPrivado discovers data elements that are being collected, processed, or shared in the code.\n")
+    println(s"DATA ELEMENTS  |  ${sourceNameIdMap.size} |")
+    println(s"THIRD PARTY    |  ${uniqueThirdParties.size} |")
+    println(s"ISSUES         |  ${violationSize} |")
+    println("\n---------------------------------------------------------")
+    println(s"${sourceNameIdMap.size} DATA ELEMENTS")
+    println("Here is a list of data elements discovered in the code along with details on data flows to third parties, databases and leakages to logs.")
+
+    var count = 0;
+    sourceNameIdMap.foreachEntry((sourceId, sourceName) => {
+      count = count + 1
+
+      Console.println(s"\n${RESET}${WHITE}${BOLD}${count}. ${sourceName.toUpperCase()}${RESET}")
+
+      if (thirdPartySourceMap.contains(sourceId)) {
+        Console.println(s"\t${RESET}${YELLOW}Sharing${RESET}         ->  ${thirdPartySourceMap(sourceId).toList.mkString(", ")}")
```

Comment:
To get desired result change the mkString line to for all below  as well --->

thirdPartySourceMap(sourceId).toList.map(item => "\""+item+"\"").mkString(", ")

Reply:
No, we don't want to add the additional quotes. Not sure why prefix is getting added.

Reply:
If we want the result to be --> "google.com", "facebook.com", "mongo.com"
then you can refer to my comment to achieve that. Else ignore